code/index.html

```diff
@@ -1,30 +1,174 @@
+
+    <header class="image-header">
+
+      <!-- Nav -->
+      <nav>
+        <ul>
+          <li><a href="#">Prices</a></li>
+          <li><a href="#">About us</a></li>
+          <li><a href="#">Contact us</a></li>
+        </ul>
+      </nav>
+
+      <h1>Vancouver running tours</h1>
+      <p>A unique sightseeing experience</p>
+      <a class="header-button" href="#">Book now</a>
+
+    </header>
+
+    <!-- Main -->
+
+    <main class="main">
+      
+      <!-- Signup form -->
+
+      <section class="signup-form-section">
+        
+        <form
+          class="signup-form"
+          action="https://httpbin.org/anything"
+          method="post"
+        >
+          <h2>Sign up for our newsletter</h2>
+
+          <label for="fname">First name:</label>
+          <input
+            type="text"
+            name="fname"
+            id="fname"
+            placeholder="Your first name"
+            maxlength="20"
+            required
+          />
+
+          <label for="lname">Last name:</label>
+          <input
+            type="text"
+            name="lname"
+            id="lname"
+            placeholder="Your last name"
+            maxlength="30"
+            required
+          />
+
+          <label for="email">Email:</label>
+          <input 
+            type="email" 
+            name="email" 
+            id="email" 
+            required 
+          />
+
+          <label for="age">Age:</label>
+          <input 
+            type="number" 
+            name="age" 
+            id="age" 
+            min="15" 
+            max="110" 
+            step="1" 
+          />
+
+          <label for="experience">Running experience:</label>
+            <select name="experience" id="experience">
+              <option value="beginner">Beginner</option>
+              <option value="intermediate">Intermediate</option>
+              <option value="advanced">Advanced</option>
+            </select>
+          </label>
+
+          <div class="radio">
+            <p>Your preferred distance</p>
+
+            <input 
+              type="radio" 
+              name="distance" 
+              id="5k" 
+              value="5k" 
+              checked 
+            />
+            <label for="5k">5K</label>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+            <input
+              type="radio"
+              name="distance"
+              id="10K"
+              value="10K"
+            />
+            <label for="10K">10K</label>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+            <input 
+              type="radio" 
+              name="distance" 
+              id="marathon" 
+              value="marathon" 
+            />
+            <label for="marathon">Marathon</label>
+
+          </div>
+
+          <input class="submitbutton" type="submit" value="Sign up" />
```

Comment:
One input tag are structured in pieces and the other one in the same row.

I personally prefer the last one but both are good as long as it is constent 👍